resources/locales/en/translation.json

```diff
@@ -6,12 +6,19 @@
   "Der Untertitel benennt präzise, um welche Wahl es sich handelt&#46;": "the subtitle gives a precise name of what election is concerned&#46;",
   "% Hürde": "threshold in %",
   "Falls eine Prozenthürde exisitert, bitte Zahl eintragen&#46;": "if a percentage hurdle exists, please enter the number&#46;",
+  "Daten mit Fehlermarge": "Data with error margin",
+  "Beschriftung der Legende": "Legend labels",
+  "Beschriftung des besten Werts in der Legende": "Best guess label",
+  "Beschriftung des Fehlerbereichs": "Error margin label",
   "Parteien": "parties",
   "Partei": "party",
   "Parteiname": "party name",
   "Parteifarbe": "party color",
   "Farbcode": "color code",
   "CSS Klassenname": "CSS class name",
+  "mit Fehlerbereich": "with error margin",
```

Comment:
I think we should consistently use "Fehlerbereich" or "Fehlermarge" and not use both

Reply:
done